front/pages/api/w/[wId]/data_sources/managed.ts

```diff
@@ -316,21 +316,6 @@ async function handler(
 
       const dataSource = dataSourceView.dataSource;
 
-      // For each data source, we create two views:
-      // - One default view in its associated vault
-      // - If the data source resides in the system vault, we also create a custom view in the global vault until vault are released.
-
-      if (dataSource.vault.isSystem()) {
```

Comment:
Wait, we still need to create the global vault data source view no?

Reply:
Nope we don't, the view will be created when an admin add data from this new connection.

Reply:
kk